scripts/pre-commit.sh

```diff
@@ -149,7 +149,7 @@ echo "Downloading e2e test dependencies"
 ./scripts/download-e2e-binaries.sh
 
 CREATE_INSECURE_REGISTRY=y CONFIGURE_INSECURE_REGISTRY_HOST=y \
-    KIND_TAG="v1.18.4" ./scripts/create-clusters.sh
+    KIND_TAG="v1.19.2" ./scripts/create-clusters.sh
```

Comment:
There is only a [v1.19.1 tag available](https://hub.docker.com/r/kindest/node/tags?page=1&name=v1.19) here so please use that.
```suggestion
    KIND_TAG="v1.19.1" ./scripts/create-clusters.sh
```

Reply:
Done